lib/media/play_rate_controller.js

```diff
@@ -94,10 +107,10 @@ shaka.media.PlayRateController = class {
     this.timer_.stop();
 
     shaka.log.v1('Changing effective playback rate to', this.rate_);
```

Comment:
This log should be after you get the `rate` variable, and should use it instead.
E.g. so it says we are setting the effective playback rate to 0 if we have `isBuffering_` set.

Reply:
Thanks @theodab  just corrected